jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java

```diff
@@ -26,7 +26,7 @@
 @CucumberOptions(
         features = "classpath:features"
         , glue = {"com.epam.jdi.bdd", "cucmberTests"}
-        //, tags = {"@menu"}
+        , tags = {"@MultiDropDown"}
```

Comment:
should be commented in bdd branch, now it's only for local purposes

Reply:
fixed